galley/pkg/config/analysis/README.md

```diff
@@ -177,6 +177,14 @@ You can use `istioctl experimental analyze` to run all analyzers, including your
 make istioctl && $GOPATH/out/linux_amd64/release/istioctl experimental analyze
 ```
 
+### 6. Write a user-facing documentation page
+
+Each analysis message needs to be documented for customers. This is done by introducing a markdown file for
+each message in the istio.io repo in the content/en/docs/reference/config/analysis directory. You create
+a subdirectory with the code of the error message, and add a `index.md` file that contains the
+full description of the problem with potential remediation steps, examples, etc. See the existing
+files in that directory for examples of how this is done.
```

Comment:
change files to subdirectories? It sounds like there should be a `analysis` subdirectory with subdirectories per analysis message (it doesn't seem to exist yet). It sounds like you are asking them to see the other existing subdirectories under `analysis` and their `index.md` files.